packages/xo-web/src/common/icon.js

```diff
@@ -6,7 +6,7 @@ import React from 'react'
 const Icon = ({ icon, size = 1, color, fixedWidth, ...props }) => {
   props.className = classNames(
     props.className,
-    icon !== undefined ? `xo-icon-${icon}` : 'fa', // Without icon prop, is a placeholder.
+    !icon ? 'fa' : `xo-icon-${icon}`, // Misaligned problem modification: if no icon or null, apply 'fa' 
```

Comment:
Explicit test:
```suggestion
    icon == null ? 'fa' : `xo-icon-${icon}`, // Misaligned problem modification: if no icon or null, apply 'fa' 
```

Reply:
Also, try to keep your changes minimal, unless I'm mistaken, this should be working too and reveal that the problem was that `null` was not correctly handled:

```suggestion
    icon != null ?  `xo-icon-${icon}` : 'fa', // Misaligned problem modification: if no icon or null, apply 'fa' 
```